UefiCpuPkg/PiSmmCpuDxeSmm/X64/PageTbl.c

```diff
@@ -432,7 +467,16 @@ SmmInitPageTable (
     // Add pages to page pool
     //
     FreePage = (LIST_ENTRY *)AllocatePageTableMemory (PAGE_TABLE_PAGES);
-    ASSERT (FreePage != NULL);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (FreePage == NULL) {
+      ASSERT (FreePage != NULL);

```

Comment:
Should we continue here?

Reply:
That seems to be a good idea. Can you confirm the latest code matches what you were thinking?

Reply:
Yep, thanks!